inc/options/class-wpseo-options.php

```diff
@@ -446,7 +446,13 @@ public static function reset_ms_blog( $blog_id ) {
 	public static function save_option( $wpseo_options_group_name, $option_name, $option_value ) {
 		$options                 = self::get_option( $wpseo_options_group_name );
 		$options[ $option_name ] = $option_value;
-		update_option( $wpseo_options_group_name, $options );
+
+		if ( isset( self::$option_instances[ $wpseo_options_group_name ] ) && self::$option_instances[ $wpseo_options_group_name ]->multisite_only == true ) {
```

Comment:
any specific reason I might not be aware of to not use `===` here? `multisite_only == true`

Reply:
@afercia Oh I missed this, it should indeed be using `===`.